pkg/vetter/util/util.go

```diff
@@ -481,3 +482,24 @@ func ConvertHostnameToFQDN(hostname string, namespace string) (string, error) {
 	// need to return Fully Qualified Domain Name
 	return hostname + "." + namespace + KubernetesDomainSuffix, nil
 }
+
+// Extract status port from the cmd arguments for a given container, or
+//  default that is 15020 (as per Istio 1.1 doc, global.proxy.statusPort
+//  at https://istio.io/docs/reference/config/installation-options-changes/
+func ProxyStatusPort(container corev1.Container) uint32 {
+	var statusPort uint32 = 15020
```

Comment:
do you think we should return default even if it is not passed in as argument?

Reply:
Need to add unit tests for this util

Reply:
Mmm - maybe it is an 'error' not to have this defined as part of the Args to start with? It might be better if we assume all ports are subject to mTLS if there is nothing found - I could go either way really...

Reply:
Added unit test.